src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/TemplateObjectWrapperTests.kt

```diff
@@ -0,0 +1,92 @@
+package org.vaccineimpact.orderlyweb.tests.unit_tests
+
+import com.nhaarman.mockito_kotlin.doReturn
+import com.nhaarman.mockito_kotlin.mock
+import freemarker.ext.beans.StringModel
+import freemarker.template.SimpleHash
+import freemarker.template.SimpleSequence
+import freemarker.template.TemplateBooleanModel
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.pac4j.core.profile.CommonProfile
+import org.vaccineimpact.orderlyweb.ActionContext
+import org.vaccineimpact.orderlyweb.Serializer
+import org.vaccineimpact.orderlyweb.app_start.TemplateObjectWrapper
+import org.vaccineimpact.orderlyweb.controllers.web.HomeController.IndexViewModel
+import org.vaccineimpact.orderlyweb.controllers.web.Serialise
+import org.vaccineimpact.orderlyweb.models.Artefact
+import org.vaccineimpact.orderlyweb.models.ArtefactFormat
+import org.vaccineimpact.orderlyweb.models.Report
+import org.vaccineimpact.orderlyweb.models.ReportVersionDetails
+import org.vaccineimpact.orderlyweb.test_helpers.TeamcityTests
+import java.time.Instant
+
+class TemplateObjectWrapperTests : TeamcityTests()
```

Comment:
Minor point, but I wonder if this is an integration test rather than a unit test, since it relies on knowledge of other classes like ReportVersionDetails. So these tests will break if the defn of those classes change. Can see why you want to test that this works though!